Makefile

```diff
@@ -51,6 +51,7 @@ lint: .venv
 apidocs: .venv
 	@$(VENV_PATH)/bin/pip install --editable .
 	@$(VENV_PATH)/bin/pip install -U -r ./docs-requirements.txt
+	rm -rf docs/_build
```

Comment:
Making sure all the docs are rebuilt every time. (For CI this is not important because it is run in a new runner anyways, more for local apidocs generation)